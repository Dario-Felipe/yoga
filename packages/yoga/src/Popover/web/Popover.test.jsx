import React from 'react';
import { render } from '@testing-library/react';
import { Info } from '@gympass/yoga-icons';

import { ThemeProvider, Popover } from '../..';

describe('<Popover />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <Popover
          title="Making wellbeing universal"
          description="Try moving to another spot. Your new favorite activity could be arround the corner."
          width={265}
        >
          <Info width={26} height={26} />
        </Popover>
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
