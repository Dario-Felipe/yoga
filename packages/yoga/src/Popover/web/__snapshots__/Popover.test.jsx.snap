// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Popover /> should match snapshot 1`] = `
.c0 {
  position: relative;
}

<div>
  <div
    class="c0"
  >
    <button
      style="all: unset;"
      type="button"
    >
      <svg
        height="26"
        width="26"
      />
    </button>
  </div>
</div>
`;
